bitcoin/src/pow.rs

```diff
@@ -400,7 +415,6 @@ impl U256 {
     }
 
     /// Returns the least number of bits needed to represent the number.
-    #[cfg_attr(all(test, mutate), mutate)]
```

Comment:
These shouldn't be removed please, they are used when mutation testing.

Reply:
Ops, I think this came from @junderw PR, I merged and forgot to add them, thanks for the catch